tools/convert_datasets/uav123/uav123_infos.txt

```diff
@@ -0,0 +1,124 @@
+The format of each line in this txt is (video_path,annotation_path,start_frame_id,end_frame_id)
```

Comment:
We already have this [file](https://github.com/open-mmlab/mmtracking/blob/master/tools/convert_datasets/uav123/uav123_info.txt)

Reply:
The old information format is  anno_name,anno_path,video_path,start_frame,end_frame,
now the new informat format is  video_path,annotation_path,start_frame_id,end_frame_id

Reply:
change the original filename to `uav123_infos_deprecated`.